raiden-ts/src/channels/utils.ts

```diff
@@ -74,6 +75,10 @@ export function channelAmounts(channel: Channel) {
       partnerCapacity: Zero32,
       partnerOnchainUnlocked: Zero32,
       partnerUnlocked: Zero32, // total of off & onchain unlocked
+      ownTotalWithdrawable: Zero32,
+      ownWithdrawable: Zero32,
```

Comment:
On the `ChannelBalances` interface, they're described in comments. I'll add a nice JSDoc to it to make it clearer. The total is not the maximum that could be withdrawn (this would be the `ownWithdrawable`), but instead the maximum value acceptable for `setTotalWithdraw`, taking into account balances and etc. `Withdrawable` value is (as seen in code) `TotalWithdrawable - withdraw (on-chain persisted)`